examples/elixir_spherical_advection_covariant.jl

```diff
@@ -0,0 +1,98 @@
+###############################################################################
+# DGSEM for the linear advection equation on the cubed sphere
+###############################################################################
```

Comment:
This problem is based on the standard Williamson cosine bell case, but adapted to be a Gaussian to make it $C^\infty$. I agree that it would be good to use the same case for the Cartesian and covariant form, but is it alright with you if we make both of them standard "geophysical" cases like this one?

Reply:
That sounds reasonable!

Reply:
Ok, sounds good - I will address in the next commit.

Reply:
Addressed in [78f8593](https://github.com/trixi-framework/TrixiAtmo.jl/pull/31/commits/78f859398e5b7a50ec4395177a3a655a31e305b6).

Reply:
Ok, thanks! It makes a lot of sense to use `ShallowWaterEquations3D` instead of the 3D Euler equations for the spherical advection test. 

I wouldn't remove the covariant advection equation. It's cleaner to have a separate equation than an equation modified by a source term... And you already invested some time in writing that module.